base/threadingconstructs.jl

```diff
@@ -8,6 +8,26 @@ export threadid, nthreads, @threads, @spawn,
 
 Get the ID number of the current thread of execution. The master thread has
 ID `1`.
+
+# Examples
+```julia-repl
+julia> Threads.threadid()
+1
+
+julia> Threads.@threads for i in 1:10
```

Comment:
Reduce the number of iterations a bit here (and below) to save some precious vertical space?

Reply:
Hopefully addressed!